trailblazer/store/crud/delete.py

```diff
@@ -20,6 +20,7 @@ def delete_analysis_jobs(self, analysis: Analysis) -> None:
         session: Session = get_session()
         for job in analysis.jobs:
             session.delete(job)
+        session.commit()
```

Comment:
Do we want this here, or should we perform the commit in the CLI/Endpoint level? If possible I would prefer to have it on the same level for all

Reply:
Me too, but I don't like cluttering the cli and api with these commits and I don't want to spend more time thinking about it :sweat_smile: I'm leaving it for now as is, if someone (probably me in the future) feel like they have time and energy to refactor it

Reply:
The session/transaction management should not be done in the controller or CLI layer either. I think one way to go is to use the `teardown_request` decorator https://flask.palletsprojects.com/en/3.0.x/api/#flask.Flask.teardown_request. But I'm pretty over dealing with this. Feel free to pick it up if you are motivated :muscle: